docs/dev_guide/contents/newcomers.rst

```diff
@@ -191,6 +191,25 @@ If you get stuck or want help, just `ask here`_!
 .. _sunpy repository: https://github.com/sunpy/sunpy
 .. _ask here: https://openastronomy.element.io/#/room/#sunpy:openastronomy.org
 
+Setting up a development environment using Vagrant (optional)
```

Comment:
I think this file should be reverted back to normal. I am worried it might confuse people but if people are aware of vagrant, hopefully they will spot the file in the repo.

Reply:
Done 👍🏻